README.yaml

```diff
@@ -49,6 +49,46 @@ introduction: |-
   |         2               |           65 536         | 
   |         3               |         16 777 216       |
   
+  If `nested-matrices-count` input is `1` output `matrix` would be json formatted string with a following structure
+  
+  ```yaml
+  {
+    "include": [matrix items]
+  }
+  ```
+
+  If `nested-matrices-count` input is `2` output `matrix` would be json formatted string with a following structure
+  
+  ```yaml
+  {
+    "include": [{
+      "name": "group name",
+      "items": {
+        "include": [matrix items]
+      } ## serialized as string
+    }]
+  }
+  ```
+  
+  If `nested-matrices-count` input is `3` output `matrix` would be json formatted string with a following structure
+  
+  ```yaml
+  {
+    "include": [{
+      "name": "group name",
+      "items": [{
+        "name": "chunk 256 range name",
+        "include": [
+          "items": {
+            "include": [matrix items] ## serialized as string
+          }
+        ]
+      }] ## serialized as string
+
+      } ## serialized as string
+    }]
+  }
+  ```
   
   > [!WARNING]  
   > Restrict concurrency to avoid DDOS GitHub Actions API and get restriction on your account.
```

Comment:
```suggestion
  > Make sure you [restrict concurrency](https://docs.github.com/en/actions/using-jobs/using-concurrency) of your jobs to avoid DDOS'ing the GitHub Actions API, which might cause restrictions to be applied to your account.
```